src/patternfly/components/Label/styles.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../patternfly-utilities";
+
+.pf-c-label {
+  // Component
+  --pf-c-label--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label--LineHeight: var(--pf-global--LineHeight--md);
```

Comment:
I think you can remove this, the [line height in _base.scss](https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/_base.scss#L11) is already `--pf-global--LineHeight--md`

Reply:
okay sounds good